src/applications/personalization/profile/util/direct-deposit.js

```diff
@@ -90,7 +90,7 @@ export class DirectDepositClient {
     this.recordAnalyticsEvent(payload);
   }
 
-  recordCNPEvent({ status, method, extraProperties = {} }) {
+  recordCNPEvent({ status, method = 'GET', extraProperties = {} }) {
```

Comment:
Now that you've set the default parameter to `GET`, can you delete the other instances where you call `recordCNPEvent`?